src/components/NewFooter/Menu.style.tsx

```diff
@@ -75,12 +100,17 @@ export const Section = styled.div`
   }
 `;
 
-export const SectionHeader = styled.h2`
+export const SectionHeader = styled.h2<{ desktopOnly?: boolean }>`
   ${fonts.regularBookMidWeight};
-  color: ${COLOR_MAP.GREY_3};
+  color: ${props => props.theme.palette.megaMenu.gray};
```

Comment:
Maybe this should be named according to it's usage like the other `megaMenu` properties? `sectionHeaderColor` maybe?

Reply:
its also used to color the arrow icons, so i just kept it general